build/Dockerfile.adapter

```diff
@@ -0,0 +1,19 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+
+ENV OPERATOR=/usr/local/bin/keda-adapter \
+    USER_UID=1001 \
+    USER_NAME=keda
+
+COPY build/_output/bin/keda-adapter ${OPERATOR} 
+
+COPY build/bin /usr/local/bin
```

Comment:
I think this line is not needed, right? It fails for me on both images on a `make build`. Removing it from both resolves the issue.

Reply:
Actually I think that folder is just missing the `user_setup` script.

Reply:
Good catch, thanks. `.gitignore` was too strict and prevented pushing those files. There's so much files, so I haven't noticed that.  I'll fix it in a few.

Reply:
@ahmelsayed should be fixed now, PTAL